pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/GetClinicalOrdersForTissuesRoute.java

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.dsm.route;
+
+import com.google.gson.Gson;
+import com.google.gson.annotations.SerializedName;
+import lombok.Getter;
+import org.broadinstitute.dsm.db.dao.mercury.ClinicalOrderDao;
+import org.broadinstitute.dsm.db.dto.mercury.ClinicalOrderDto;
+import org.broadinstitute.dsm.security.RequestHandler;
+import spark.Request;
+import spark.Response;
+
+import java.util.List;
+
+/**
+ * Consumes a list of tissue id ints from a post and returns
+ * a map where the keys are the tissue ids and the values
+ * are the clinical orders for the tissue.
+ */
+public class GetClinicalOrdersForTissuesRoute extends RequestHandler {
```

Comment:
Down the road, we do plan on making a query for orders accessible to the frontend.  This route is not yet used, so it does not have any tests, but I've kept it here so we can use it in a subsequent sprint.